### ISSUE TITLE
Verify that start/stop/clean modes works with configure for core, passive, active, all

### ISSUE THREAD

**Issue by cprafullchandra** [CyberReboot/vent#59]
References: #47 #54 #56

**Comment by cglewis** [comment]
`plugins` shouldn't have a start/stop option since they're on-demand containers.  not sure I understand what you're verifying here?

**Comment by cprafullchandra** [comment]
Bad word choice on my part, meant start/stop services/containers (core, passive-only, active-only, all services)

**Comment by cprafullchandra** [comment]
To be verified for Cores & Passive/Active & All:
* Start/Stop works with no enabling/disabling: start should start all core services, and stop should stop all core services.
* The first start/stop should take into account configurations in core.templates/modes.templates and only start non-disabled cores.
* When core.templates or modes.template is configured after starting/stopping/cleaning, any containers that were running and are now disabled should be stopped. Opting for not removing them.
* If a core is disabled and stopped, cleaning should remove the container and not re-create it.
* If a core was disabled and is now enabled, start cores should create + start the core as usual.

**Comment by cglewis** [comment]
```
Disabling a running Core should kill and remove it
 -  Disabled in modes.template [FAILED]
 - Disabled in core.template [FAILED]
```
really? which commit ID are you testing against?  I thought I fixed this with #63

**Comment by cglewis** [comment]
can you add numbers in front of these so we can reference them easily?

**Comment by cprafullchandra** [comment]
Will do. Commit ID I was testing against: https://github.com/cprafullchandra/vent/commit/6159977ede0c7f232d7c8a28433023f6bb939ebc

**Comment by cglewis** [comment]
interesting. perhaps there's a scenario I didn't account for.

**Comment by cglewis** [comment]
current thinking:

core:
 - `4-6, 16-17, 20`: we should fix and add issues for them for 0.2.0
 - `15, 18`: hold off for now, need to think through the consequences of that

passive/active:
 - `39-42, 47-48`: we should fix and add issues for them for 0.2.0
 - `37-38, 43-44`: hold off for now, need to think through the consequences of that

all:
 - `4-6, 10-12`: should already work, i think
 - `13-15`: if they don't work we should fix and add issues for them for 0.2.0

visualization:
 - should probably test that too at some point in the future (not now)

**Comment by cprafullchandra** [comment]
Verification done. Creating issue #96.

After #96 is closed, should revisit verification.

**Comment by cglewis** [comment]
@cprafullchandra status of this now that #96 is closed?

**Comment by cprafullchandra** [comment]
@cglewis Will look into it!

**Comment by cprafullchandra** [comment]
@cglewis Testing from scratch now

**Comment by cprafullchandra** [comment]
New bug: All > Start All now causes rq-worker to break and stop.

**Comment by cglewis** [comment]
@cprafullchandra couldn't reproduce that.
